### DISCUSSION TITLE
Things that should be moved to pymc-experimental

### DISCUSSION THREAD

**Discussion by ricardoV94** [pymc-devs/pymc3#5508] [General]
Now that we have [pymc-experimental](https://github.com/pymc-devs/pymc-experimental) (almost) up and running, we should decide what parts of the library should be moved there (if any)!

The goal is to separate components that are properly tested (theoretically, empirically and, perhaps as important, code-wise), from more experimental/flaky components.

***

I am adding a purely gut-feeling derived list below. Feel free to argue for items to be removed or added from the list:

Modules:
* Bart module. Moved #5566 

Step methods:
* MLDA
* SGMCMC. Issue https://github.com/pymc-devs/pymc/issues/5580

Distributions:
* Wishart
* WishartBartlett
* PolyaGamma

Some esoteric VI methods?

**Comment by twiecki** [comment]
Agree with all of these. SGMCMC can probably be removed all-together.

**Reply by fonnesbeck** [reply]
Yeah, maybe do a PR that removes it and see if anyone objects. There aren't many (only 1 I could find) questions on Discourse about it, so that is probably a sign that it is not being used.

**Reply by fbarfi** [reply]
I would not remove polyagamma 
this has opened a new front for logistic bayesian models as it was demonstrated in the 2013 original papers. I think that aesara is incorporating a version of the logistic Bayesian model using polyagamma. Thanks.

**Reply by ricardoV94** [reply]
@fbarfi we are not removing but moving to a separate library `pymc-experimental`, which should be straightforward to use together with PyMC

**Comment by fbarfi** [comment]
Ricardo 👍  thanks for the clarification.
